compiler/rustc_mir_build/src/build/matches/simplify.rs

```diff
@@ -44,12 +44,36 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
         candidate: &mut Candidate<'pat, 'tcx>,
     ) -> bool {
         // repeatedly simplify match pairs until fixed point is reached
+        debug!("simplify_candidate(candidate={:?})", candidate);
```

Comment:
```suggestion
        debug!(?candidate, "simplify_candidate");
```

or put

```
#[tracing::instrument(skip(self))]
```

onto the function